tools/meson/Makefile

```diff
@@ -20,8 +20,11 @@ define Host/Compile
 endef
 
 define Host/Install
+	$(INSTALL_DIR) $(STAGING_DIR_HOST)/bin
+	$(HOST_BUILD_DIR)/packaging/create_zipapp.py $(HOST_BUILD_DIR) --interpreter $(STAGING_DIR_HOST)/bin/$(PYTHON) --outfile $(STAGING_DIR_HOST)/bin/meson
```

Comment:
can you elaborate on this script? Do we happen to have that in tree or is this part of meson?

Reply:
it's part of meson. Hence the HOST_BUILD_DIR